src/pages/mint/index.js

```diff
@@ -119,34 +119,42 @@ export const Mint = () => {
               <Input
                 type="text"
                 onChange={(e) => setTitle(e.target.value)}
-                placeholder="title"
+                label="TITLE"
+                value={title}
               />
 
               <Input
                 type="text"
                 onChange={(e) => setDescription(e.target.value)}
-                placeholder="description"
+                label="DESCRIPTION"
```

Comment:
also, including new text will force me to chase everyone who contributed towards the translations. why do we need this?

Reply:
I'm happy to chase down the translations - I should just look at who commited translation changes right? I wasn't able to work out how the translations work actually so thought maybe they won't being used yet.